infrastructure/quick-deploy/localhost/variables.tf

```
@@ -302,18 +302,18 @@ variable "fluent_bit" {
 }
 
 # Extra configuration
-variable "extra_conf" {
-  description = "Add extra configuration in the configmaps"
+variable "configurations" {
+  description = ""
   type = object({
-    compute = optional(map(string), {})
-    control = optional(map(string), {})
-    core    = optional(map(string), {})
-    log     = optional(map(string), {})
-    metrics = optional(map(string), {})
-    polling = optional(map(string), {})
-    worker  = optional(map(string), {})
+    core    = optional(any, {})
+    control = optional(any, {})
+    compute = optional(any, {})
+    worker  = optional(any, {})
+    polling = optional(any, {})
+    log     = optional(any, {})
+    metrics = optional(any, {})
+    jobs    = optional(any, {})
```

I would go with `[]` for consistency

```suggestion
    core    = optional(any, [])
    control = optional(any, [])
    compute = optional(any, [])
    worker  = optional(any, [])
    polling = optional(any, [])
    log     = optional(any, [])
    metrics = optional(any, [])
    jobs    = optional(any, [])
```